### ISSUE TITLE
[BUG] Events Feed fail to pull from Webtools events; only displaying Big 10 Athletics category [ver. 5.0.55]

### ISSUE THREAD

**Issue by knguyenjahiel** [rokwire/illinois-app#3663]
**Describe the bug**
Events Feed is only displaying Big 10 Athletics category and events created in testing session. It is not displaying events from Webtools.

**To Reproduce**
1. Events >> Event Feed
2. You will only see events in the Big 10 Athletics category and events created from testing session. 

**Expected behavior**
Events in other categories, created in Webtools, should display.

 - Device: Pixel 6
 - OS: Android 13
 - Version 5.0.55 
 
**Additional context**
This issue might be related to @vburgett's [#3606](https://github.com/rokwire/illinois-app/issues/3606) Git issue ("Events under Exhibits not pulling in Webtools Exhibition events").

**Comment by sandeep-ps** [comment]
Hi, Dobromir, After I saw this issue created, I did some testing to check if there if there is a data issue. The events data is present in the backend and for the most part, also displayed as you saw. In my testing, after not using my test phone for some time (maybe a few days), I saw that the events (and possibly other content) were not displayed. I was shown as logged in, and after closing the instance of the app, I could see the events and all other content too. I agree that this may not be easily reproducible.
